flask_common/crypto.py

```diff
@@ -58,13 +68,17 @@ def aes_encrypt_iv(key, data, iv):
 
 # Verify + decrypt using no IV or provided IV. Pass empty string for no IV.
 # Note: You should normally use aes_decrypt()
-def aes_decrypt_iv(key, data, iv):
+def aes_decrypt_iv(key, data, iv, extracted_version=None):
     aes_key = key[:AES_BLOCK_SIZE]
     hmac_key = key[AES_BLOCK_SIZE:]
     sig_size = HMAC_DIGEST_SIZE
     cipher = data[:-sig_size]
     sig = data[-sig_size:]
     if hmac.new(hmac_key, iv + cipher, HMAC_DIGEST).digest() != sig:
+        if extracted_version:
+            return aes_decrypt(
```

Comment:
Should add a comment here explaining we are falling back and trying to decrypt assuming it is  legacy non-versioned data